index.js

```diff
@@ -231,7 +239,7 @@ class Plugin extends PluginMiniAccounts {
     const primary = data.protocolData[0]
     if (primary.protocolName === 'machinomy') {
       const payment = JSON.parse(primary.data.toString())
-      console.log("GOT PAYMENT", payment)
```

Comment:
We should remove console.log statements. Could make it a `debug` one if you think it's useful to have this type of information printed